lnrpc/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.20.4-buster
+FROM golang:1.21.0-bullseye
```

Comment:
If we want `bullseye`, we need to bump the compiler/formatter versions to:
```
  protobuf-compiler='3.12.4*' \
  clang-format='1:11.0*'
```

For `bookworm` it would be:
```
  protobuf-compiler='3.21.12*' \
  clang-format='1:14.0*'
```

See https://packages.debian.org/bookworm/clang-format and https://packages.debian.org/bookworm/protobuf-compiler.